cfme/utils/appliance/__init__.py

```diff
@@ -676,10 +676,6 @@ def ui_port(self):
         else:
             raise Exception("Unknown scheme {} for {}".format(parsed_url.scheme, store.base_url))
 
-    @cached_property
```

Comment:
Looks like attr set in `IPAppliance.__init__`, safe to remove

Reply:
yep, that's the reason to remove it

Reply:
Yeah, #5810 should get rid of all these cached properties once merged :)

Reply:
I remove this property only because I have to have correct schema in server.address